### PR TITLE
Add code of conduct report

### PR DESCRIPTION
**Pull request by williln**

### CODE REVIEW

File: _posts/2017-08-24-conduct-report.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+- An attendee reported that a speaker's presentation made them feel uncomfortable and excluded. Conference organizers reviewed the presentation and determined that it did not violate the Code of Conduct. 
+- A conference attendee reported that someone not affiliated with the conference was verbally harassing them and someone else, also not affiliated with the conference, in the park close to where our opening reception was held. At the time the report was made, the harasser had already left the area. Conference staff assisted the attendee in contacting local law enforcement and asked them if they needed other help. They declined, and the harasser was not seen at our event again. 
+
+We're grateful to those who reported possible Code of Conduct violations to us, and encourage any attendee who experienced harassment at our conference, witnessed someone else being harassed, or has any questions to email the Code of Conduct response team at [coc@djangocon.us](coc@djangocon.us).
```
Comment (jefftriplett) [inline _posts/2017-08-24-conduct-report.md]: It should be [coc@djangocon.us](mailto:coc@djangocon.us) to fix the html checker error. The rest looks good!

**Review by jefftriplett — CHANGES_REQUESTED** [review]

**Comment by williln** [comment]
@jefftriplett OK, Travis is happy (well, still unhappy with the url for one of the slides but that's being addressed separately). Thanks!

**Comment by jefftriplett** [comment]
@williln LGTM, thank you! I think Andrew has contacted them about his account being messed up. I'll push an update to fix it post merge.

**Review by jefftriplett — APPROVED** [review]
🎉

**Pull request merged**